pkgs/by-name/ca/cargo-leptos/no-vendored-openssl.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/Cargo.toml b/Cargo.toml
+index e705c3c..8289018 100644
+--- a/Cargo.toml
++++ b/Cargo.toml
+@@ -56,7 +56,7 @@ base64ct = { version = "1.7.3", features = ["alloc"] }
+ swc = "23.0"
+ swc_common = "9.0"
+ shlex = "1.3"
+-cargo-generate = { version = "0.23.3", features = ["vendored-openssl"] }
++cargo-generate = "0.23.3"
```

Comment:
We don't have to patch this, simply setting `env.OPENSSL_NO_VENDOR = 1;` is enough.